website/src/components/navbar/navbar.tsx

```diff
@@ -42,6 +44,7 @@ export default async function Navbar({ lang, country }: DefaultParams) {
 					],
 				},
 				{ title: transparency, href: transparencyHref },
+				{ title: selection, href: selectionHref },
```

Comment:
I think we should probably also show a dropdown for the transparency section. No need to do it in this PR though, I'll do it in another one.

Reply:
it's okay, I've got it!